assets/app/styles/_log.less

```diff
@@ -100,14 +90,16 @@
   }
 }
 .log-line-number {
-  .flex(@columns: 0 0 60px); // set width of number column
+  .unselectable();
+  border-right: 1px lighten(@log-bg-color, 10%) solid;
+  min-width: 60px;
```

Comment:
ideally, a short log would reserve the minimal space for single or double-digit lines, and a long log would grow the left column to fit 4,5,6 digit numbers